#Number of orders: 1
#Number of shipments: 1
#Insufficiently many products: No
#Flipped products: No
#In-process order update: No
#Dropped products: No
#In-process order interruption: Yes
#Part flipping: No
#Faulty products: No
#Moving obstacles: No
#Sensor blackout: No
#2 arms advantageous: No
#Time limit: No
options:
  insert_models_over_bins: true
  insert_models_over_shelves: true
  belt_population_cycles: 0
  spawn_extra_models: false
  model_type_aliases:
    order_part1: piston_rod_part_blue
    order_part2: gear_part_green
    order_part3: gear_part_red

shelf_layout:
  row_1: ['base',0,'collar','base'] #shelves: 3, 4, 5
  row_2: [0, 'pipe','base','base']   #shelves: 6, 7, 8
  row_3: ['base','pipe',0,'collar'] #shelves: 9, 10, 11

orders:
  order_0:
    announcement_condition: time
    announcement_condition_value: 0.0
    shipment_count: 1
    destinations: [agv1]
    products:
      part_0:
        type: order_part1
        pose:
          xyz: [0.1, -0.2, 0]
          rpy: [0, 0, 0]
      part_1:
        type: order_part1
        pose:
          xyz: [-0.1, -0.2, 0]
          rpy: [0, 0, 0]
      part_2:
        type: order_part3
        pose:
          xyz: [0.15, 0.15, 0]
          rpy: [0, 0, 0]
      part_3:
        type: order_part2
        pose:
          xyz: [-0.15, 0.15, 0]
          rpy: [0, 0, 0]
      part_4:
        type: order_part2
        pose:
          xyz: [0.0, 0.15, 0]
          rpy: [0, 0, 0]

  order_1:
    announcement_condition: unwanted_products
    announcement_condition_value: 1
    shipment_count: 1
    destinations: [agv2]
    products:
      part_0:
        type: order_part2
        pose:
          xyz: [0.1, -0.2, 0]
          rpy: [0, 0, 0]
      part_1:
        type: order_part2
        pose:
          xyz: [-0.1, -0.2, 0]
          rpy: [0, 0, 0]

models_over_bins:
   bin3:
     models:
       gear_part_green:
         xyz_start: [0.2, 0.2, 0.0]
         xyz_end: [0.4, 0.4, 0.0]
         rpy: [0, 0, 0]
         num_models_x: 2
         num_models_y: 2
   bin4:
     models:
       piston_rod_part_blue:
         xyz_start: [0.1, 0.2, 0.0]
         xyz_end: [0.4, 0.5, 0.0]
         rpy: [0, 0, 'pi/4']
         num_models_x: 2
         num_models_y: 3

models_over_shelves:
  shelf1:
    models:
      gear_part_red:
        xyz_start: [-1.5, -0.5, 0.0]
        xyz_end: [-1.0, -0.5, 0.0]
        rpy: [0, 0, 0]
        num_models_x: 5
        num_models_y: 1


time_limit: -1
